x-pack/plugin/security/cli/src/main/java/org/elasticsearch/xpack/security/cli/CertificateTool.java

```diff
@@ -167,8 +167,9 @@ public static void main(String[] args) throws Exception {
 
     static final String CA_EXPLANATION =
         "    * All certificates generated by this tool will be signed by a certificate authority (CA).\n" +
-            "    * The tool can automatically generate a new CA for you, or you can provide your own with the\n" +
-            "         -ca or -ca-cert command line options.";
+            "    * The certificates can be self signed using the -self-signed command line option.\n" +
+            "         Alternatively the tool can automatically generate a new CA for you, or you can " +
+            "         provide your own with the -ca or -ca-cert command line options.";
```

Comment:
How about something like: 

```suggestion
"    * All certificates generated by this tool will be signed by a certificate authority (CA), unless" +
            "    the --self-signed command line option is specified.\n" +
            "         The tool can automatically generate a new CA for you, or you can " +
            "         provide your own with the -ca or -ca-cert command line options.";
```

I think it reads better but I'll let Lisa tell us :)